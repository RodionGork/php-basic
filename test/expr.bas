print "2+3*5=", 2+3*5
print "3*(1+2)/(5-1)", 3*(1+2)/(5-1)
print "5*-2=", 5*-2

rem power has reverted order!
print "2^3^2=", 2^3^2, "(should be 2^9=512, not 8^2=64)"

print "5 > 3 + 1", 5 > 3 + 1
print "3 * 2 < 5", 3 * 2 < 5
print "5 > 3 and 7 > 5", 5 > 3 and 7 > 5
print "5 > 3 and 3 > 5", 5 > 3 and 3 > 5
print "5 > 3 or 3 > 5", 5 > 3 or 3 > 5
