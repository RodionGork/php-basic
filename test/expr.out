2+3*5=17
3*(1+2)/(5-1)2.25
5*-2=-10
2^3^2=512 (should be 2^9=512, not 8^2=64)
5 > 3 + 1 gives 1
3 * 2 < 5 gives 0
5 > 3 and 7 > 5 gives 1
5 > 3 and 3 > 5 gives 0
5 > 3 or 3 > 5 gives 1
